llvm/include/llvm/CodeGen/SelectionDAGNodes.h

```diff
@@ -381,6 +381,7 @@ struct SDNodeFlags {
   bool NoUnsignedWrap : 1;
   bool NoSignedWrap : 1;
   bool Exact : 1;
+  bool Disjoint : 1;
```

Comment:
I thought this was unioned with the wrap flags. I guess we don't have enough of these bits to matter yet